console/jsre/deps/web3.js

```diff
@@ -5422,6 +5426,13 @@ var methods = function () {
         outputFormatter: formatters.outputBlockFormatter
     });
 
+    var getHeader = new Method({
+        name: 'getHeader',
+        call: headerCall,
+        params: 1,
+        inputFormatter: [formatters.inputBlockNumberFormatter]
```

Comment:
I have a question. Do you know what `function (val) { return !!val; }` is? `getBlock` has another parameter.

Reply:
I was also curious when I wrote it. 
`getBlock` has one more parameter `fullTX` which type is boolean. 
I think `function (val) { return !!val; }` is ethereum style's boolean formatter...

Reply:
Ah, thanks for explanation.